Generator/DoctrineCrudGenerator.php

```diff
@@ -78,7 +78,7 @@ public function generate(BundleInterface $bundle, $entity, ClassMetadataInfo $me
 
         $this->generateControllerClass($forceOverwrite);
 
-        $dir = sprintf('%s/Resources/views/%s', $this->rootDir, str_replace('\\', '/', strtolower($this->entity)));
+        $dir = sprintf('%s/Resources/views/%s', $this->bundle->getPath(), str_replace('\\', '/', strtolower($this->entity)));
```

Comment:
I was suggested this:
$dir = sprintf('%s/Resources/views/%s', $this->bundle->getPath(), str_replace('\\', '/', ucfirst($this->entity)));